packages/plugin/tests/require-description.spec.ts

```diff
@@ -197,5 +218,25 @@ ruleTester.runGraphQLTests<[RequireDescriptionRuleConfig]>('require-description'
       options: [OPERATION],
       errors: [{ message: 'Description is required for `query`.' }],
     },
+    {
+      code: 'type Query { user(id: String!): User! }',
+      options: [{ operationFieldDefinition: true }],
+      errors: [{ message: 'Description is required for `Query.user`.' }],
+    },
+    {
+      code: 'type Query { users: [User!]! }',
+      options: [{ operationFieldDefinition: true }],
+      errors: [{ message: 'Description is required for `Query.users`.' }],
+    },
```

Comment:
```suggestion
```
one test for `Query` is enough, instead include 1 test with a root type that is named differently than `Query`/`Mutation` or `Subscription`

Reply:
@B2o5T I added a test case for a root type with [a field as a Correct case](https://github.com/B2o5T/graphql-eslint/pull/1303/commits/6c36d2778c4500ec65a13a0ff4ab8fd0869d73af#diff-a303039c21671d709870c86d53e96a474dff89cd481f292e6fb278ee868bb233R99-R101). Is this what you had in mind? If I misread you I'd be happy to try again